### ISSUE TITLE
python blown-up by tkinter demo code

### ISSUE THREAD

**Issue by Bernd54Albrecht** [lawsie/guizero#332]
Hello again, I tried the tkinter demo code "Spinbox". Now I cannot use Python at all, not IDLE and not Thonny. new update/upgrade shoed also errors. I am about to format the µSD-Card. See attached photos.

Quotation: "Using tkinter widgets in guizero
You can add tk widgets into your guizero app using the add_tk_widget method of App, Window and Box.

In this example, we are adding the tkinter widget Spinbox into a guizero App:

from guizero import App, Text
from tkinter import Spinbox

app = App()
text = Text(app, text="A Spinbox widget")

spinbox = Spinbox(from_=0, to=10)
app.add_tk_widget(spinbox)

app.display()
End of Quotation
![IMG_1463](https://user-images.githubusercontent.com/6961239/63166089-a5d87b00-c02d-11e9-835b-5c8b123313ae.JPG)
![IMG_1464](https://user-images.githubusercontent.com/6961239/63166090-a5d87b00-c02d-11e9-9855-77b808d02021.JPG)

**Comment by martinohanlon** [comment]
This is not an issue with guizero. 

I have on occasion seen this issue with IDLE and a reboot has fixed it.

You will find information on getting help using Python in the guizero documentation  https://lawsie.github.io/guizero/gettinghelp/

Raising issues on github is not an effective way of getting support as only myself and @lawsie monitor these issues.

**Comment by Bernd54Albrecht** [comment]
Roger. However, it was your example code that caused this major problem Bernd Albrecht -------- Originalnachricht --------Betreff: Re: [lawsie/guizero] python blown-up by tkinter demo code (#332)Von: martinohanlon An: lawsie/guizero Cc: Bernd54Albrecht ,Author Closed #332.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/lawsie/guizero/issues/332?email_source=notifications\u0026email_token=ABVDQV5KVWBCULR7BFTBYH3QE26P7A5CNFSM4IMGZYLKYY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGOTC637MA#event-2562572208",
"url": "https://github.com/lawsie/guizero/issues/332?email_source=notifications\u0026email_token=ABVDQV5KVWBCULR7BFTBYH3QE26P7A5CNFSM4IMGZYLKYY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGOTC637MA#event-2562572208",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by martinohanlon** [comment]
The example code does work on Raspbian. 

![image](https://user-images.githubusercontent.com/2329501/63177780-8e56bd80-c040-11e9-979d-6cc9f2e6d5ac.png)

Sorry I cant help with your issue.

**Comment by Bernd54Albrecht** [comment]
Not closed!

Houston, we've got a problem! I.e. You've got a problem!

Brandnew installation, update, upgrade, sudo pip3 install guizero --upgrade

Nevertheless: Demo program "Spinbox" leads to a crash of Python on my RasPi
4 B!

I saw the program work on my RasPi 3B+ with the previous Raspbian Duster.

Your turn
Greetings
Bernd54Albrecht

Am Fr., 16. Aug. 2019 um 17:12 Uhr schrieb martinohanlon <
notifications@github.com>:
